pkg/sdkserver/alpha/localsdk.go

```diff
@@ -14,20 +14,49 @@
 
 package alpha
 
-import "agones.dev/agones/pkg/sdk/alpha"
+import (
+	"context"
+
+	"agones.dev/agones/pkg/sdk/alpha"
+)
 
 var _ alpha.SDKServer = LocalSDKServer{}
 
 // LocalSDKServer is the local sdk server implementation
-// for alpha features
+// for alpha features.
 type LocalSDKServer struct{}
 
-// NewLocalSDKServer is a constructor for the alpha local SDK Server
+// NewLocalSDKServer is a constructor for the alpha local SDK Server.
 func NewLocalSDKServer() *LocalSDKServer {
 	return &LocalSDKServer{}
 }
 
-// Close tears down all the things
+// Close tears down all the things.
 func (l *LocalSDKServer) Close() {
 	// placeholder in case things need to be shut down
 }
+
+// PlayerConnect should be called when a player connects
```

Comment:
nit: end comment sentence with a period.

(I was going to let this slide until I found other things to comment on. As long as you are fixing those, this should be an easy addition)